rj_gameplay/rj_gameplay/tactic/goalie_tactic.py

```diff
@@ -113,61 +190,80 @@ def get_requests(self, world_state: rc.WorldState,
                     role_requests[self.receive_se] = [
-                        role.RoleRequest(role.Priority.HIGH, True, self.role_cost)
+                        role.RoleRequest(role.Priority.HIGH, True,
+                                         self.role_cost)
                     ]
                 else:
+                    # clear
                     # if ball has been stopped already, chip toward center field
-                    self.pivot_kick_se.skill.target_point = np.array([0.0, 6.0])
+                    self.pivot_kick_se.skill.target_point = np.array(
+                        [0.0, 6.0])
                     role_requests[self.pivot_kick_se] = [
-                        role.RoleRequest(role.Priority.HIGH, True, self.role_cost)
+                        role.RoleRequest(role.Priority.HIGH, True,
+                                         self.role_cost)
                     ]
             else:
-                if ball_speed > 0 and np.dot(towards_goal, world_state.ball.vel) > 0.3:
+                if best_utility is block_utility:
                     # if ball is moving and coming at goal, move laterally to block ball
                     # TODO (#1676): replace this logic with a real intercept planner
-                    goalie_pos = world_state.our_robots[world_state.goalie_id].pose[
-                                 :2] if world_state.goalie_id is not None else np.array([0., 0.])
-                    self.move_se.skill.target_point = get_block_pt(world_state, goalie_pos)
+                    goalie_pos = world_state.our_robots[
+                        world_state.
+                        goalie_id].pose[:
+                                        2] if world_state.goalie_id is not None else np.array(
+                                            [0., 0.])
+                    self.move_se.skill.target_point = get_block_pt(
+                        world_state, goalie_pos)
                     self.move_se.skill.face_point = world_state.ball.pos
                     role_requests[self.move_se] = [
                         role.RoleRequest(role.Priority.HIGH, True,
                                          self.role_cost)
                     ]
-                else:
-                    # else, track ball normally
+                elif best_utility is track_utility:
+                    # track ball normally
                     self.move_se.skill.target_point = get_goalie_pt(
                         world_state)
                     self.move_se.skill.face_point = world_state.ball.pos
                     role_requests[self.move_se] = [
                         role.RoleRequest(role.Priority.HIGH, True,
                                          self.role_cost)
                     ]
+                elif best_utility is pass_utility:
+                    # TODO : under current system, should assign pass tactic here
+                    pass
+
+        # what is this? and should I keep it?
```

Comment:
This is some more hacky stuff from comp2021. Keep it for now, but try removing it in testing and seeing what happens.